app/Http/Controllers/IpmaWarningController.php

```diff
@@ -17,7 +19,14 @@ class IpmaWarningController extends Controller
      */
     public function index(): JsonResponse
     {
-        $data = IpmaWarning::query()->where('is_active', 1)->get();
-        return response()->collection($data, new IpmaWarningSerializer());
+        $data = ['data' => []];
+
+        if (Cache::has('ipma_warnings')) {
+            $ipmaWarnings           = Cache::get('ipma_warnings');
+            $ipmaWarningsCollection = (new Collection($ipmaWarnings, new IpmaWarningSerializer()));
+            $data                   = new Document($ipmaWarningsCollection);
+        }
+
+        return response()->json($data);
```

Comment:
Use `response()->collection()` like you had previously.

Reply:
hummm, that method only allows EloquentCollection ... :/ i tried before pick this solution.